packages/language-server/src/prisma-fmt/referentialActions.ts

```diff
@@ -1,23 +1,9 @@
-import execa from 'execa'
+import prismaFmt from '@prisma/prisma-fmt-wasm'
 
 export default function referentialActions(
-  execPath: string,
   text: string,
   onError?: (errorMessage: string) => void,
 ): string[] {
-  const result = execa.sync(execPath, ['referential-actions'], { input: text })
-
-  if (result.exitCode !== 0) {
-    const errorMessage =
-      "prisma-fmt error'd during getting available referential actions.\n"
```

Comment:
Lost error message and complete error handling here actually @tomhoule

Reply:
https://github.com/prisma/language-tools/issues/965

applied labels so it doesn't get lost but I intend to do this today.